src/modules/75-cf/pages/feature-flags-detail/targeting-rules-tab/TargetingRulesTab.tsx

```diff
@@ -179,7 +210,24 @@ const TargetingRulesTab = ({
       initialValues={initialValues}
       validate={values => validate(values)}
       onSubmit={values => {
-        saveChanges(values)
+        if (initialValues.state === 'on') {
+          if (initialValues.onVariation != values.onVariation) {
```

Comment:
would you be able to combine these nested `if` statements? Either way it's fine
```suggestion
          if (initialValues.state === 'on' && initialValues.onVariation != values.onVariation) {
```

Reply:
No unfortunately not because the else is on the second if not the first